src/views/layout.njk

```diff
@@ -3,12 +3,60 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <title>GOV.UK Frontend</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="/public/css/govuk-frontend.css">
   <!--[if IE 8]>
   <link rel="stylesheet" href="/css/govuk-frontend-oldie.css">
   <![endif]-->
+  <style>
+    /* Injected into the preview page, that is then put in an iframe */
+    .app-iframe-in-component-preview {
+      margin-top: 10px;
+      margin-bottom: 0;
+    }
+
+    .app-iframe-in-component-preview:before {
```

Comment:
I'm guessing this is nuking the breakpoint thing? Definitely worth a comment.

Reply:
Good shout